dev-requirements.txt

```diff
@@ -9,4 +9,6 @@ grpcio-tools==1.26.0
 protobuf==3.13.0
 six==1.14.0
 tox == 3.15.0
-cloudevents >= 0.3.0
\ No newline at end of file
+cloudevents >= 0.3.0
```

Comment:
should this be set to 1.0? @youngbupark

Reply:
yes seems like 1.x is released from aug. 

This needs to be upgraded too.

https://github.com/dapr/python-sdk/blob/bccfcf72271bb2357ae1b7326adb06442a9b089b/ext/dapr-ext-grpc/setup.cfg#L25